assets/vue/convos-connection-editor.vue

```diff
@@ -128,8 +133,10 @@ module.exports = {
     },
     updateForm: function(connection) {
       var url = connection ? connection.url.parseUrl() : null;
+      var on_connect = connection ? connection.on_connect_commands.join("\n") : "";
```

Comment:
I would rather keep the variable as-is and join it when displaying it inside convos-connection-editor.vue.